otx/cli/tools/build.py

```diff
@@ -0,0 +1,69 @@
+"""OTX building command 'otx build'.
+
+This command allows you to build an OTX workspace, provide usable backbone configurations,
+and build models with new backbone replacements.
+"""
+# Copyright (C) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import argparse
+import os
+
+from otx.cli.builder import Builder
+
+SUPPORTED_TASKS = ("CLASSIFICATION", "DETECTION", "INSTANCE_SEGMENTATION", "SEGMENTATION")
+
+
+def parse_args():
+    """Parses command line arguments."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--task", help=f"The currently supported options: {SUPPORTED_TASKS}.")
+    parser.add_argument("--workspace-root", help="The path to use as the workspace.")
+    parser.add_argument("--model", help="Input OTX model config file (e.g model.py).", default=None)
+    parser.add_argument("--backbone", help="Enter the backbone configuration file path or available backbone type.")
+    parser.add_argument("--save-backbone-to", help="Enter where to save the backbone configuration file.", default=None)
+    parser.add_argument("--root", help="A root dir where templates should be searched.", default=".")
```

Comment:
I think root could be confusing because it can mean other things in CS. how about to use other words? (For example, template-dir or template-root?)

Reply:
This means the root path of otx. Would otx-root be appropriate? I'm not sure.

Reply:
I see. then why user needs to set this argument? As far as I know, it can get own package path by itself. does this code consider finding template in other OTX directory?

Reply:
This `root` argment is because we need to know the otx root to build the template (when create workspace).
Of course, most users think that they will create a workspace in the otx root, but even in a non-root place, if the root is given as a separate input, it helps to create it there.

Reply:
Edited as per your suggestion. could you please check it?

Reply:
thanks for reflecting!